docker-compose/tomcat-manager/docker-compose.yml

```diff
@@ -0,0 +1,241 @@
+version: '3.3'
+services:
+  postgres:
+    image: 'postgres:9'
```

Comment:
Might be good to externalise the version of Postgres, Keycloak, etc.

Also, we should use the newest version of Postgres.